sdks/go/container/build.gradle

```diff
@@ -46,13 +46,7 @@ golang {
 }
 
 docker {
-  String repositoryRoot
-  if (rootProject.hasProperty(["docker-repository-root"])) {
-    repositoryRoot = rootProject["docker-repository-root"]
-  } else {
-    repositoryRoot = "${System.properties["user.name"]}-docker-apache.bintray.io/beam"
-  }
-  name "${repositoryRoot}/go:latest"
+  name containerImageName("go")
   files "./build/"
```

Comment:
Note that with this setup we are building the boot binary multiple times, once per container.

Reply:
Not sure I understand what you mean. We are only building one container image, like before.

Reply:
We have the same golang definition to build the boot.go binary in sdks/java/container, sdks/go/container, and sdks/python/container.

Reply:
They are different boot.go files.